nestkernel/connection_manager.h

```diff
@@ -94,6 +94,7 @@ class ConnectionManager
    * 'target' a token array with GIDs of target neuron.
    * If either of these does not exist, all neuron are used for the respective entry.
    * 'synapse_model' name of the synapse model, or all synapse models are searched.
+   * 'synapse_label' label (int) of the synapse, or all synapses are searched.
```

Comment:
You write `int` here, but it is a `long`, at least on the C++ level. Or is this SLI documentation?